foxy/distribution.yaml

```diff
@@ -4380,6 +4380,21 @@ repositories:
       url: https://github.com/ros-visualization/rqt_robot_steering.git
       version: dashing-devel
     status: maintained
+  rqt_runtime_monitor:
+    doc:
+      type: git
+      url: https://github.com/ros-visualization/rqt_runtime_monitor.git
+      version: rolling
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/ros2-gbp/rqt_runtime_monitor-release.git
+      version: 0.5.8-1
+    source:
+      type: git
+      url: https://github.com/ros-visualization/rqt_runtime_monitor.git
```

Comment:
@arne48 please add a LICENSE file at the root of the source respository that matches the one listed in the `package.xml`

Reply:
This is a legacy package so we won't block on this, but it would be good practice to include.